src/repository/noteRelations.repository.ts

```diff
@@ -47,4 +47,22 @@ export default class NoteRelationsRepository {
   public async getParentNoteIdByNoteId(noteId: NoteInternalId): Promise<number | null> {
     return await this.storage.getParentNoteIdByNoteId(noteId);
   }
+
+  /**
+   * Delete all note ralations contains noteId
+   *
+   * @param noteId - id of the current note
+   */
+  public async deleteNoteRelationsByNoteId(noteId: NoteInternalId): Promise<boolean> {
+    return await this.storage.deleteNoteRelationsByNoteId(noteId);
```

Comment:
note can have only one relation as a child, rename method to `deleteNoteRelationByNoteId()` and change description please